src/main/java/org/folio/fqm/service/EntityTypeInitializationService.java

```diff
@@ -74,7 +91,8 @@ public void initializeEntityTypes() throws IOException {
           return objectMapper.readValue(
             resource
               .getContentAsString(StandardCharsets.UTF_8)
-              .replace("${tenant_id}", folioExecutionContext.getTenantId()),
+              .replace("${tenant_id}", folioExecutionContext.getTenantId())
+              .replace("${central_tenant_id}", finalCentralTenantId),
```

Comment:
Do we need to do this in the tenant management service, too? That'd get us consistent behavior between liquibase and entity type definitions, but I'm not sure if that's necessary or not

Reply:
I didn't think that was necessary at the moment since there's nowhere within the liquibase scripts where we actually needed the central tenant id, and I can't think of a situation where we would need it. It shouldn't be a big deal to add later if we need to though.

Reply:
good catch...the potential dependency concerns scare me lol